src/glog/logging.h.in

```diff
@@ -1574,9 +1599,13 @@ GLOG_EXPORT void FlushLogFilesUnsafe(LogSeverity min_severity);
 // messages is sent.  If base_filename is "", it means "don't log this
 // severity".  Thread-safe.
 //
+#if defined(GLOG_OS_WINDOWS) && defined(UNICODE)
```

Comment:
I think we should provide public API functionality unconditionally as otherwise this might confuse users why their code does not compile without `UNICODE` definition. Ideally, the support for wide strings should be accessible cross-platform.